INSTALL.md

```diff
@@ -137,11 +118,22 @@ subjects:
   - apiGroup: rbac.authorization.k8s.io
     kind: User
     name: $ADMIN_USERNAME
+EOF
 ```
 
-### Container registry credentials
+## Dependencies
+
+### Cert-Manager
+
+[Cert-Manager](https://cert-manager.io) allows us to create internal automatically certificates within the cluster. Follow the [instructions](https://cert-manager.io/docs/installation/) to install the latest version.
 
-Use the following command to create a `Secret` that Korifi will use to connect to your container registry:
+### Kpack
+
+[Kpack](https://github.com/pivotal/kpack) powers our source-to-image build process. Follow the [instructions](https://github.com/pivotal/kpack/blob/main/docs/install.md) to install the latest version.
```

Comment:
I know you just moved this line up, but after re-reading it I think it might be confusing to new users. Maybe something like this would be more clear:

```suggestion
[Kpack](https://github.com/pivotal/kpack) is used to build runnable applications from source code using [Cloud Native Buildpacks](https://buildpacks.io/). Follow the [instructions](https://github.com/pivotal/kpack/blob/main/docs/install.md) to install the latest version.
```